### PR TITLE
Add float_precision parameter to to_pomdp_file()

### PR DESCRIPTION
**Pull request by Hororohoruru**
As discussed in #28, a particular case where the uniform belief over states is a periodic number (0.083333333 in this case) created an error when running SARSOP on the `.pomdp` file created by `to_pomdp_file()`. This was because the float to str conversion defaults to 6 decimals. 

Adding a parameter to control for this precision in `to_pomdp_file()` fixed the issue.

### CODE REVIEW

File: README.rst
Comment (zkytony) [inline README.rst:1]: I understand this change is due to merging from the main branch, but this change shouldn't be part of this PR as it is not relevant.

Comment (Hororohoruru) [reply README.rst:1]: Sorry, didn't notice! I just rebased to remove that commit.
